strings/src/main/res/values/strings.xml

```diff
@@ -9,7 +9,7 @@
     the specific language governing permissions and limitations under the
     License.
 -->
-<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation" tools:locale="en">
+<resources xmlns:tools="http://schemas.android.com/tools" xmlns:oat="http://schema.getodk.org/odkAndroidTools" tools:ignore="MissingTranslation" tools:locale="en">
```

Comment:
I wasn't sure about if we needed an actual like URL for this schema seeing as it's just an experiment for the moment. @lognaturel I think you have a bit more experience with XML schemas than me, so it would be interesting to see what your take is.

Reply:
It’s really just an opaque identifier so this looks great to me! Oats are tasty.